packages/sanity/src/core/studio/components/navbar/resources/helper-functions/types.ts

```diff
@@ -0,0 +1,58 @@
+import {Image, PortableTextBlock} from '@sanity/types'
+
+interface WelcomeVideo {
+  link?: Link
+  videos?: Video[]
+}
+
+interface Video {
+  description?: PortableTextBlock[]
+  image?: Image
+  title?: string
+  youtube?: YouTube[]
+}
+
+interface Link {
+  url?: string
+  title?: string
+}
+
+interface YouTube {
+  duration?: number
+  url?: string
+  videoId?: string
+}
+
+interface Resource {
+  sectionArray?: Section[]
+  title?: string
+}
+
+export interface Section {
+  _key: string
+  sectionTitle?: string
+  items?: (InternalAction | ExternalLink)[]
+}
+
+interface Item {
+  _key: string
+  title?: string
+}
+
+interface ExternalLink extends Item {
+  _type: 'externalLink'
+  url?: string
+}
+
+interface InternalAction extends Item {
+  _type: 'internalAction'
+  type?: InternalActionType
+}
+
+type InternalActionType = 'show-welcome-modal'
+
+export interface ResourcesResponse {
```

Comment:
💭 Should this have tsdoc comment, if it's internal then I would add the suggestion

```suggestion
/**
 * @hidden
 * @beta */
export interface ResourcesResponse {
```